hub/hub_service.bal

```diff
@@ -42,21 +40,29 @@ http:Service healthCheckService = service object {
         }
         healthcheck:HealthCheckResp diskSpace = {status: diskSpaceStatus, details: {diskSpaceMetaData}};
 
-        //kafka
-        string kafkaStatus = "DOWN";
-        kafka:TopicPartition[]|kafka:Error producerResult = conn:statePersistProducer->getTopicPartitions(config:REGISTERED_WEBSUB_TOPICS_TOPIC);
```

Comment:
this looks lite, can we use this in consolidator ?

Reply:
done in java

Reply:
resolved